pages/tutorial/gui.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page-fullwidth
+title: "Graphical Interface"
+meta_title: "The Alchemist's Graphical Interface"
+subheadline: "Learn the basics!"
+teaser: "Alchemist is equipped with a Java graphical interface, and here is how to use it."
+permalink: "pages/tutorial/gui/"
+comments: true
+#categories:
+#    - design
+header:
+   image_fullwidth: "header_mol.png"
+---
+
+## Starting Alchemist in graphical mode
+
+### Prerequisites
+
+* A working graphical environment
+* A Java 8 (or newer) runtime environment
+
+### Download and installation
+
+Alchemist does not require any installation procedure. It is delivered in form of a runnable jar file, that can be retrieved from [the Alchemist release page on Github](https://github.com/AlchemistSimulator/alchemist/releases). The file name should match ``alchemist-redist-VERSION.jar`` where ``VERSION`` is a [SemVer](http://semver.org/spec/v2.0.0.html) version.
```

Comment:
Please make this consistent with the fact that we offer a way to configure the simulator via Gradle